vendor/README.md

```diff
@@ -48,6 +48,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **Boo:** [Shammah/boo-sublime](https://github.com/Shammah/boo-sublime)
 - **Brainfuck:** [Drako/SublimeBrainfuck](https://github.com/Drako/SublimeBrainfuck)
 - **Brightscript:** [cmink/BrightScript.tmbundle](https://github.com/cmink/BrightScript.tmbundle)
+- **BuildStream:** [BuildStream/buildstream](https://gitlab.com/BuildStream/buildstream)
```

Comment:
This looks incorrect. How did you add this? Did you read the contribution guidelines?